### ISSUE TITLE
Missing statistics in TaxBrain difference table

### ISSUE THREAD

**Issue by martinholmer** [ospc-org/ospc.org#770]
@martinholmer said in [Tax-Calculator issue 1737](https://github.com/open-source-economics/Tax-Calculator/issues/1737#issuecomment-350418216):
> Part of the problem in the TaxBrain difference table is that even though Tax-Calculator is computing a "Percentage Increase in After-Tax Expanded Income" statistic, TaxBrain is not yet displaying it. Again, this is a known problem with TaxBrain that we hope to fix soon.

I thought this was a "known problem", but I can't seem to find an issue that describes this as a high-priority enhancement to TaxBrain.

**Comment by martinholmer** [comment]
Actually there are **two** missing statistics in the TaxBrain difference table.
Here is the specification of the table in Tax-Calculator `utils.py`:
```
DIFF_TABLE_COLUMNS = ['count',
                      'tax_cut',
                      'perc_cut',
                      'tax_inc',
                      'perc_inc',
                      'mean',
                      'tot_change',
                      'share_of_change',
                      'perc_aftertax',
                      'pc_aftertaxinc']

DIFF_TABLE_LABELS = ['All Tax Units',
                     'Tax Units with Tax Cut',
                     'Percent with Tax Cut',
                     'Tax Units with Tax Increase',
                     'Percent with Tax Increase',
                     'Average Tax Change',
                     'Total Tax Difference',
                     'Share of Overall Change',
                     'Change as % of After-Tax Income',
                     '% Change in After-Tax Income']
```
TaxBrain displays only the first eight (of these ten) statistics.  It needs to display all ten statistics.

**Comment by MaxGhenis** [comment]
Are there still plans to add "Percentage Increase in After-Tax Expanded Income" to TaxBrain difference tables?

**Comment by martinholmer** [comment]
There are (in PolicyBrain 1.5.0) still missing statistics in the difference table.  The distribution table includes all the statistics generated by Tax-Calcalutor, but the difference table is missing two statistics: the consumption value of benefits and the percentage change in after-tax expanded income.

What's the schedule for fixing this bug?

@hdoupe @GoFroggyRun

**Comment by hdoupe** [comment]
Thanks for the bug report @martinholmer. I will address the schedule for this in a separate issue.

**Comment by martinholmer** [comment]
@hdoupe, the bug report in issue #770 has been open for almost 11 months.
What's the timeframe for fixing these problems with the difference tables?

**Comment by hdoupe** [comment]
@martinholmer thanks for bringing this back up. It is likely that this was resolved or at least, the power to resolve it was handed back to the Tax-Calculator project by the backwards-incompatible changes made in PR #921. The fixes from #921 are no longer on the test app because I needed to test out v1.7.2_rc1. I hope to have a better version of Tax-Calculator running on the webapp in the next month or two.
